src/content/docs/analyzer/import-sorting.mdx

```diff
@@ -113,6 +113,40 @@ import "../polyfills/array/flatMap";
 import { functionThatUsesFlatMap } from "./utils.js";
 ```
 
+## Side effect imports
+
+A [side effect imports](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/import#forms_of_import_declarations) are import statements that don't import any name:
+
+```js
+import "./global.js"
+```
+
+Since it is hard to determine which side effect the module brought,
+The import organizer creates a group for every side effect imports.
+This means that imports that appear before and imports that appear after a side effect imports are independently sorted.
```

Comment:
When you fixed the issue, you talked about each single side-effect module being a single group. That explanation was simple and effective. I would use that explanation.

The explanation proposed is a bit difficult to follow, especially because we talked about side-effect modules until now, and we focus on the groups before and after the side-effects.